caffe2/python/onnx/backend.py

```diff
@@ -1018,7 +726,7 @@ def prepare(cls, model, device='CPU', **kwargs):
         # Build the C++ backend
         # TODO: build a predictor that supports GPU
         #       And for RNN nets, we need to avoid adding init_net
-        if device == 'CPU' and not rnn_nodes:
+        if False: #device == 'CPU' and not rnn_nodes:
```

Comment:
Is this too hacky? Do we want to directly remove this branch?

Reply:
I forgot to remove this debug string...